rpcs3/Crypto/utils.cpp

```diff
@@ -9,13 +9,15 @@
 #include <stdio.h>
 #include <time.h>
 #include "Utilities/StrUtil.h"
-#include "Utilities/span.h"
 #include "Utilities/File.h"
 
 #include <memory>
 #include <string>
 #include <string_view>
 
+#include <span>
+//#include "Utilities/span.h"

```

Comment:
No point in comment.

Reply:
i commented it out instead of removing it just to be safe :)

Reply:
It just confuses who reads it, kind of TMI of unrelated events in code history.

Reply:
fair .. i also just realised i basically redid the entirety of an existing commit because im an idiot ://